jekyll/_cci2/configuration-reference.md

```diff
@@ -144,6 +144,7 @@ docker | Y <sup>(1)</sup> | List | Options for [docker executor](#docker)
 machine | Y <sup>(1)</sup> | Map | Options for [machine executor](#machine)
 macos | Y <sup>(1)</sup> | Map | Options for [macOS executor](#macos)
 shell | N | String | Shell to use for execution command in all steps. Can be overridden by `shell` in each step (default: See [Default Shell Options](#default-shell-options))
+parameters | N | List | A list of parameters that need to be passed as part of the request when performing this operation
```

Comment:
@thomasr8 It's a `Map` like the `parameters` fields in the other 2.1 root fields (`commands`, etc.). I guess it's based on the same code.

Reply:
Ahhhh....thanks for the correction. I will go ahead and make the change in the docs this week. Thank you so much for helping with this.